shared/constants/chat2/meta.tsx

```diff
@@ -292,7 +292,7 @@ export const inboxUIItemToConversationMeta = (
     i.convSettings && i.convSettings.minWriterRoleInfo ? i.convSettings.minWriterRoleInfo.cannotWrite : false
   const conversationIDKey = Types.stringToConversationIDKey(i.convID)
   let pinnedMsg: PinnedMessageInfo | undefined
-  if (i.pinnedMsg) {
+  if (i.pinnedMsg && state) {
```

Comment:
`state` should be passed here really, but we do call this in some other random loading calls which never involve pinned message so lets just make this optional